internal/ethapi/api.go

```diff
@@ -1251,7 +1251,7 @@ func (s *PublicBlockChainAPI) rpcOutputBlock(b *types.Block, inclTx bool, fullTx
 		"timestamp":        (*hexutil.Big)(head.Time),
 		"transactionsRoot": head.TxHash,
 		"receiptsRoot":     head.ReceiptHash,
-		"validators":       hexutil.Bytes(head.Validators),
+		"validators":       hexutil.Bytes(head.Validators.CurrentEpoch),//TODO gerui: ask what should be here!
```

Comment:
What should be the new rpc output look like now that I have two lists in "validators"? @wjrjerome  @liam-lai

Reply:
changed this line comment to `//omit head.Validators.NextEpoch on purpose`

Reply:
i think u going to generate a string hex which we won't be able to recognize. at the end of day, this hexutil function just turn a variable into hex string

Reply:
the string is concatenation of addresses. E.g., two address 0x11111111111111111111111111111, 0x22222222222222222222. Then the hex string is 0x1111111111111111111111111111122222222222222222222. Do you think we need to split the string into addresses?